tutor/src/screens/my-courses/dashboard/useDisplayedOfferings.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect, useMemo } from 'react'
```

Comment:
While I also personally prefer camelcase filenames that match the component/hook they contain, everything else is dashes so we should adhere to that standard vs ending up with a mismatch.   accordingly this should probably be: `use-displayed-offerings`